### DISCUSSION TITLE
dashy as subfolder behind a nging proxy

### DISCUSSION THREAD

**Discussion by NachoParra** [Lissy93/dashy#193] [Q&A] [answered]
Hi guys,

Many thanks for this dashboard, it looks great to keep track of all the clutter I got on my local list of apps. I have a question about the setup. I want to setup Dashy inside a subfolder on my server. I have NGINX as my frontend server and I want to proxy the requests from xxxxx.yyy/dashy to the docker image, but I'm not able to do it, and I don't find any configuration field in which I can specify this route for subfolders.

Right now, the error message I got is this one:

Cannot GET /dashy/

Do you know if this is allowed? If yes, can someone tell me how to config dashy to accept a /dashy or similar in front of the URL?

Thanks a lot!

**Comment by lissy93** [comment, marked as answer]
You should be able to set the `BASE_URL` environmental variable to the path of your sub-directory :)

**Reply by NachoParra** [reply]
Thanks a lot Lissy. With this base_url is now working!! I'm trying to mark this as answer but github is not allowing me to close it.

**Reply by lissy93** [reply]
Glad it is working :)
